pytorch_lightning/loops/batch/training_batch_loop.py

```diff
@@ -334,97 +256,7 @@ def _training_step(
             if result_collection is None:
                 return
 
-        closure_loss = None
```

Comment:
reviewer: this removed block is all specific to automatic optimization and gets moved to the OptimizerLoop